release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3338,9 +3338,40 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+//4.16.24
+[id="ocp-4-16-24_{context}"]
+=== RHSA-2024:10147 - {product-title} {product-version}.24 bug fix and security update
+
+Issued: 26 November 2024
+
+{product-title} release {product-version}.24 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:10147[RHSA-2024:10147] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:10150[RHBA-2024:10150] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.16.24 --pullspecs
+----
+
+[id="ocp-4-16-24-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, if Operator Lifecycle Manager (OLM) was unable to access the secret associated with a service account, OLM would rely on the Kubernetes API server to automatically create a bearer token. With Kubernetes versions 1.22 and later, this action is no longer automatic, so with this release OLM uses the `TokenRequest` API to request a new Kubernetes API token. (link:https://issues.redhat.com/browse/OCPBUGS-44351[*OCPBUGS-44351*])
+
+* Previously, the approval mechanism for certificate signing requests (CSRs) failed because the node name and internal DNS entry for a CSR did not match in terms of character case differences. With this release, an update to the approval mechanism for CSRs skips case-sensitive checks so that a CSR with a matching node name and internal DNS entry does not fail the check because of character case differences. (link:https://issues.redhat.com/browse/OCPBUGS-44629[*OCPBUGS-44629*])
+
+* Previously, HyperShift-based ROKS clusters were unable to authenticate through the `oc login` command. The web browser displayed an error when it attepted to retrieve the the token after selecting *Display Token*. with this release, `cloud.ibm.com` and other cloud-based endpoints are no longer proxied and authentication is successful. (link:https://issues.redhat.com/browse/OCPBUGS-44277[*OCPBUGS-44277*])
+
```

Comment:
Requesting this entry be added here on behalf of https://issues.redhat.com/browse/OSDOCS-12740. Thank you!

```
* {product-title} {product-version} now supports Operator SDK 1.36.1. See xref:../cli_reference/osdk/cli-osdk-install.adoc#cli-osdk-install[Installing the Operator SDK CLI] to install or update to this latest version.
+
[NOTE]
====
Operator SDK 1.36.1 now supports Kubernetes 1.29 and uses a {op-system-base-full} 9 base image.
====
+
If you have Operator projects that were previously created or maintained with Operator SDK 1.31.0, update your projects to keep compatibility with Operator SDK 1.36.1.
+
--
* xref:../operators/operator_sdk/golang/osdk-golang-updating-projects.adoc#osdk-upgrading-projects_osdk-golang-updating-projects[Updating Go-based Operator projects]
* xref:../operators/operator_sdk/ansible/osdk-ansible-updating-projects.adoc#osdk-upgrading-projects_osdk-ansible-updating-projects[Updating Ansible-based Operator projects]
* xref:../operators/operator_sdk/helm/osdk-helm-updating-projects.adoc#osdk-upgrading-projects_osdk-helm-updating-projects[Updating Helm-based Operator projects]
* xref:../operators/operator_sdk/helm/osdk-hybrid-helm-updating-projects.adoc#osdk-upgrading-projects_osdk-hybrid-helm-updating-projects[Updating Hybrid Helm-based Operator projects]
* xref:../operators/operator_sdk/java/osdk-java-updating-projects.adoc#osdk-upgrading-projects_osdk-java-updating-projects[Updating Java-based Operator projects]
--
+
(link:https://issues.redhat.com/browse/OCPBUGS-44485[*OCPBUGS-44485*], link:https://issues.redhat.com/browse/OCPBUGS-44486[*OCPBUGS-44486*])
```